services/slack/intermediate.go

```diff
@@ -504,6 +504,45 @@ func (t *Transformer) AddAttachmentsToPost(post *SlackPost, newPost *Intermediat
 	return props, propsByteArray
 }
 
+func buildMessagePropsFromHuddle(post *SlackPost) model.StringInterface {
+	type Attachment struct {
+		ID       int    `json:"id"`
+		Text     string `json:"text"`
+		Fallback string `json:"fallback"`
+	}
+
+	type MessageProps struct {
+		Title       string       `json:"title"`
+		EndAt       int64        `json:"end_at"`
+		StartAt     int64        `json:"start_at"`
+		Attachments []Attachment `json:"attachments"`
+		FromPlugin  bool         `json:"from_plugin"`
+	}
+
+	Props := MessageProps{
```

Comment:
0/5

```suggestion
	props := MessageProps{
```

Reply:
Heads up that this broke the PR, as the references to this var need to change as well